### ISSUE TITLE
Buffer face doesn't inherit from default face 

### ISSUE THREAD

**Issue by xofyarg** [akermu/emacs-libvterm#321]
I use "InputMonoNarrow Light" as default font by setting the ":font" property of the default face, but seems vterm uses the "InputMonoNarrow" variant. Because the weight is different, it makes me feel all the texts are bold.

Not sure if this is a bug or not, please let me know if I missed anything.

**Comment by Sbozzolo** [comment]
Vterm should inherit from the default face. Can move the point to a character in vterm and press `C-u C-x =`? It will tell you what font is being used.

**Comment by xofyarg** [comment]
Yeah, that's how I know the font weights are different. My guess is vterm just set it to normal no matter what weight default face is using.

**Comment by Sbozzolo** [comment]
I think I found the bug. We have a switch to disable bold fonts, this sets the weight of a face to be normal or bold and it disregards the information about the weight in the default face.

This is in the function `render_text` in the `vterm-module.c`

Not sure how to fix, but it is a bug.
